### PR TITLE
blockcopy: update options for reuse external

### PR DESCRIPTION
**Pull request by waynesun09**
Update options for first blockcopy try on reuse external cases.

Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py [hunk 1]
```diff
@@ -457,6 +457,10 @@ def _make_snapshot():
         # Run blockcopy command
         if rerun_flag == 1:
             options1 = "--wait %s --finish --verbose" % dest_format
+            if with_blockdev:
+                options1 += " --blockdev"
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py]: It seems that the old version of virsh doesn't support '--blockdev',
Do we need to consider it?

Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcopy.py]: yeah, I'll update with check version

**Comment by will-Do** [comment]
ACK

**Comment by waynesun09** [comment]
thx, merge now

**Pull request merged**